src/DoctorHouse.Api/Controllers/GuestsController.cs

```diff
@@ -11,18 +16,29 @@ namespace DoctorHouse.Api.Controllers
     [Route("api/v1/requests/{requestId}/guests")]
     public class GuestsController : BaseApiController
     {
-        public GuestsController(IMessageExceptionFinder messageExceptionFinder) : base(messageExceptionFinder)
+        private readonly IGuestService guestService;
+
+        private readonly IMapper mapper;
+
+        private readonly IWorkContext workContext;
+        public GuestsController(
+            IMessageExceptionFinder messageExceptionFinder,
+            IGuestService guestService,
+            IMapper mapper,
+            IWorkContext workContext) : base(messageExceptionFinder)
         {
+            this.guestService = guestService;
+            this.mapper = mapper;
+            this.workContext = workContext;
         }
 
         [Authorize]
         [HttpGet]
         public IActionResult Get(int requestId)
         {
-            var models = new List<GuestModel>()
-            {
-                new GuestModel { }
-            };
+            var guests = guestService.GetAll(requestId);
+
+            var models = this.mapper.Map<IList<GuestModel>>(guests);
 
             return this.Ok(models, true, 10);
```

Comment:
Ya que este endpoint no tiene paginación, lo único que haría es cambiar está linea a algo así:

`return this.Ok(models, false, models.Count);`

Reply:
Fixed